test/remote/gateways/remote_epay_test.rb

```diff
@@ -5,12 +5,10 @@ def setup
     Base.mode = :test
 
     @gateway = EpayGateway.new(fixtures(:epay))
-
     @credit_card = credit_card('3333333333333000')
     @credit_card_declined = credit_card('3333333333333102')
-
     @amount = 100
-    @options = {order_id: '1'}
+    @options = {order_id: '1', three_d_secure: { eci: '7', xid: '123', cavv: '456', version: '2', ds_transaction_id: '798' }}
```

Comment:
we mostly never have xid and ds_transaction_id on the same request (same as before)

Reply:
eci, cavv and xid are required for 3ds-V1. The ds_transaction_id is only required when 3DS-V2 kicks in. We can handle empty values.